### PR TITLE
Update coveralls to v2

### PR DESCRIPTION
**Pull request by shibumi**
Fixes broken coveralls Github action pipeline step.

### CODE REVIEW

**Comment by shibumi** [comment]
@lukpueh  @adityasaky I have modified the coveralls setting a bit to accommodate for development.
We now accept everything above 70% and changes can have a drift of -1%.

I doubt we will move the project back to 90% again in near future. The bad value comes from the untested command line tool.

**Review by AMRALAA10 — APPROVED** [review]

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ jobs:
     - name: Test
       run: go test -covermode atomic -coverprofile='${{ matrix.os }}-${{ matrix.go-version }}.cov' ./...
     - name: Coveralls Parallel
-      uses: coverallsapp/github-action@643bc377ffa44ace6394b2b5d0d3950076de9f63
+      uses: coverallsapp/github-action@v2
```
Comment (adityasaky) [inline .github/workflows/build.yml:23]: We should probably try and pin to hash, dependabot supports those bumps.

Also, 643bc377ffa44ace6394b2b5d0d3950076de9f63 is the latest release commit AFIACT? https://github.com/coverallsapp/github-action/releases/tag/v2.3.0

So, do we need this diff?

**Review by adityasaky — COMMENTED** [review]
Thanks for fixing this @shibumi! Was the threshold change on the coveralls dashboard? If so, we should also work out a way for a few of us to have access maybe?

File: .github/workflows/build.yml (hunk 1 above)
Comment (shibumi) [reply .github/workflows/build.yml:23]: Mh, I missed this. In this case we do not need this PR. The fix on the dashboard fixed the coverall problems

**Comment by shibumi** [comment]
@adityasaky  I think you have already access. You just have to login to the coveralls UI via your GitHub account. If not, please ping me and I can take a look at the settings

**Comment by shibumi** [comment]
Closing this, because coveralls is already up to date.

**Pull request closed without merging**